adhocracy-plus/assets/scss/_layout.scss

```diff
@@ -7,6 +7,32 @@
     }
 }
 
+.container--flex {
+    display: flex;
+    justify-content: center;
+
+    @media (max-width: $breakpoint-md-down) {
+        flex-wrap: wrap;
+    }
+}
+
+.vertical-split {
+    width: 100%;
+    margin: 2rem 0;
+
+    @media (min-width: $breakpoint-md) {
+        width: 50%;
+
+        &.left {
+            margin: 2rem 1rem 2rem 5rem;
+        }
+
+        &.right {
+            margin: 2rem 5rem 2rem 1rem;
+        }
+    }
+}
+
```

Comment:
have you considered using pre-existing bootstrap here, like cols? https://getbootstrap.com/docs/4.1/layout/grid/ all designs are based on 12 col structure so it will fit for every screen size

Reply:
goodstuff, thiiink i can achieve the same with pre-existing bootstrap classes.